webpack.config.babel.js

```diff
@@ -157,7 +157,7 @@ const basePlugins = (bundle_name) => ([
     // ref: https://en.wikipedia.org/wiki/Base32
     'BASE_32_VERSION' : 'AG',
     'PRIVACY_FEATURE_ENABLED': false,
-    'LIVENESS_ENABLED': false,
+    'LIVENESS_ENABLED': true,
```

Comment:
is this intentional?

Reply:
Yes, just to allow @seewah to do the acceptance. I will switch it back to `false` when it's done.